### DISCUSSION TITLE
Can we customize the Bootstrap version on v6 ?

### DISCUSSION THREAD

**Discussion by MAR-GUEZ** [bootscore/bootscore#824] [Q&A] [answered]
Hi, 
Wondering if its possible to customize the version of bootstrap on v6 to save on speed load of unused components. Or is this something that us old people used to do thats no longer necessary?

Thanks!

**Comment by crftwrk** [comment, marked as answer]
Hi,

yes, you can simply remove/comment unused Bootstrap components in `main.scss` file https://github.com/bootscore/bootscore-child/blob/main/assets/scss/main.scss.

Solved?

**Reply by MAR-GUEZ** [reply]
Nice! Thank you!
